package.json

```diff
@@ -35,6 +35,7 @@
     "@formatjs/intl": "^2.9.9",
     "@google/model-viewer": "^3.4.0",
     "@lukso/lsp-smart-contracts": "0.15.0-rc.5",
+    "@lukso/lsp-smart-contracts-12": "npm:@lukso/lsp-smart-contracts@0.12.1",
```

Comment:
It looks like this is not used anymore, as I removed it in my PR #287

Reply:
True, if the interfaceIds are available in the latest release we no longer need to install the previous releases separately.